### PR TITLE
Add unit tests for executeHooksWithArgs.js (@wdio/sync)

### PR DESCRIPTION
**Pull request by tadashi0713**
## Proposed changes

https://github.com/webdriverio/webdriverio/issues/4292

## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) doc
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: packages/wdio-sync/tests/executeHooksWithArgs.test.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+describe('executeHooksWithArgs', () => {
+    it('multiple hooks, multiple args', async () => {
+        const hookHoge = () => {return 'hoge'}
+        const hookFuga = () => {return 'fuga'}
+        const argHoge = { hoge: 'hoge' }
+        const argFuga = { fuga: 'fuga' }
+        const res = await executeHooksWithArgs([hookHoge, hookFuga], [argHoge, argFuga])
+        expect(res).toHaveLength(2)
+        expect(res).toContain('hoge')
```
Comment (mgrybyk) [inline packages/wdio-sync/tests/executeHooksWithArgs.test.js]: Running contain here 2 times doesn't make much sense. Can you please change contain to equal to compare res with result array

**Comment by mgrybyk** [comment]
Will you be able to take a look @tadashi0713 ?

**Comment by christian-bromann** [comment]
ping @tadashi0713

**Comment by christian-bromann** [comment]
ping @tadashi0713

**Comment by mgrybyk** [comment]
I'll add the test within my PR, thanks for help!

**Pull request closed without merging**